labgrid/resource/urlresource.py

```diff
@@ -0,0 +1,10 @@
+import attr
+
+from ..factory import target_factory
+from .common import Resource
+
+@target_factory.reg_resource
+@attr.s(cmp=False)
+class URLResource(Resource):
```

Comment:
I think it would be much more useful to use a specific Resource for the Xena Devices. That way you don't have to necessarily set up named bindings, just because two of your drivers use an URLResource which are incompatible with each other.